modules/icgc-argo-workflows/prep/metrics/resources/usr/bin/rnaaln.py

```diff
@@ -29,17 +29,15 @@
 tool_fieldmap = { # 'target name' : 'original name'
     'hisat2': {
         'paired_total' : 'paired_total',
-        'unpaired_total' : 'unpaired_total',
-        'overall_alignment_rate' : 'overall_alignment_rate'
```

Comment:
`overall_alignment_rate` in `hisat2` is duplicated to `pct_reads_mapped` in `samtools_stats`. So removing it